test/test-applications/integrations/TestApplication.HttpServer/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this file needed?

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/94/commits/fb3c3e8948104661e286741005b1a9b0c258a095